src/astro/src/layouts/CareersLayout.astro

```diff
@@ -32,4 +32,7 @@ const url = Astro.url;
     {url.pathname !== "/careers" && <Join url={url} />}     
   </article>  
 </body>
+<footer>
```

Comment:
Footer element is doubly wrapped.

Reply:
I removed from the component so now only wrapped once.